src/components/Pages/PlaylistItemPage.js

```diff
@@ -46,7 +47,7 @@ const PlaylistItemPage = ({ t }) => {
 
   if (!unit) return null;
 
-  if (routeType === 'program' || !unit.collections) return <UnitPage />;
+  if (routeType === 'program' || isEmpty(unit.collections)) return <UnitPage />;
```

Comment:
isEmpty is for arrays not for objects.
In line 35 you use `Object.keys(unit.collections).length === 0` as if unit.collections is and Object.

What is the correct way?